storage/src/test/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManagerHarness.java

```diff
@@ -109,27 +108,14 @@ public void onPartitionLeadershipChanges(Set<TopicIdPartition> leaderPartitions,
             topicBasedRemoteLogMetadataManager.setRlmTopicPartitioner(remoteLogMetadataTopicPartitioner);
         }
         try {
-            waitUntilInitialized(60_000);
-        } catch (TimeoutException e) {
+            RemoteLogMetadataManagerTestUtils.waitUntilInitialized(this.remoteLogMetadataManager(), 60_000);
+        } catch (InterruptedException e) {
             throw new KafkaException(e);
         }
```

Comment:
nit: I don't think we need to catch any exception here because no matter which exception thrown, the test will fail, right?